sklearn/mixture/_gaussian_mixture_ic.py

```diff
@@ -234,19 +234,25 @@ class GaussianMixtureIC(ClusterMixin, BaseEstimator):
 
     Notes
     -----
-    This algorithm was strongly inspired by mclust, a clustering package in R
-    https://sites.stat.washington.edu/mclust/
+    This algorithm was strongly inspired by mclust [3]_,
+    a clustering package for R.
 
     References
     ----------
-    .. [1] Jeffrey D. Banfield and Adrian E. Raftery. Model-based gaussian and
-       non-gaussian clustering. Biometrics, 49:803–821, 1993.
-       https://doi.org/10.2307/2532201
-
-    .. [2] Abhijit Dasgupta and Adrian E. Raftery. Detecting features in
-       spatial point processes with clutter via model-based clustering.
-       Journal of the American Statistical Association, 93(441):294–302, 1998.
-       https://doi.org/10.2307/2669625
+    .. [1] `Fraley, C., & Raftery, A. E. (2002). Model-based clustering,
+        discriminant analysis, and density estimation.
+        Journal of the American statistical Association, 97(458), 611-631.
+       <https://doi.org/10.1198/016214502760047131>_`
```

Comment:
Arg, I forgot a space here in my previous suggestion:
```suggestion
    .. [1] `Fraley, C., & Raftery, A. E. (2002). Model-based clustering,
        discriminant analysis, and density estimation.
        Journal of the American statistical Association, 97(458), 611-631.
        <https://doi.org/10.1198/016214502760047131>_`
```

Reply:
Sorry I didn't catch that. Fixed in a new commit.